packages/components/src/ui/command.tsx

```diff
@@ -37,42 +40,55 @@ const CommandDialog = ({ children, ...props }: CommandDialogProps) => {
   )
 }
 
-const CommandInput = React.forwardRef<
-  React.ElementRef<typeof CommandPrimitive.Input>,
-  React.ComponentPropsWithoutRef<typeof CommandPrimitive.Input>
->(({ className, ...props }, ref) => (
-  <div className="flex items-center border-b px-3" cmdk-input-wrapper="">
-    <Search className="mr-2 h-4 w-4 shrink-0 opacity-50" />
-    <CommandPrimitive.Input
+type CommandInputType = React.ForwardRefExoticComponent<
+  React.ComponentPropsWithoutRef<typeof CommandPrimitive.Input> &
+    React.RefAttributes<React.ComponentRef<typeof CommandPrimitive.Input>>
+>
+
+const CommandInput: CommandInputType = React.forwardRef(
+  ({ className, ...props }, ref) => (
+    <div className="flex items-center border-b px-3" cmdk-input-wrapper="">
+      <Search className="mr-2 h-4 w-4 shrink-0 opacity-50" />
+      <CommandPrimitive.Input
+        ref={ref}
+        className={cn(
+          "flex h-11 w-full rounded-md bg-transparent py-3 text-sm outline-none placeholder:text-muted-foreground disabled:cursor-not-allowed disabled:opacity-50",
+          className,
+        )}
+        {...props}
+      />
+    </div>
+  ),
+)
+
+CommandInput.displayName = CommandPrimitive.Input.displayName
+
+type CommandListType = React.ForwardRefExoticComponent<
+  React.ComponentPropsWithoutRef<typeof CommandPrimitive.List> &
+    React.RefAttributes<React.ComponentRef<typeof CommandPrimitive.List>>
+>
+
+const CommandList: CommandListType = React.forwardRef(
+  ({ className, ...props }, ref) => (
+    <CommandPrimitive.List
       ref={ref}
       className={cn(
-        "flex h-11 w-full rounded-md bg-transparent py-3 text-sm outline-none placeholder:text-muted-foreground disabled:cursor-not-allowed disabled:opacity-50",
+        "max-h-[300px] overflow-y-auto overflow-x-hidden",
```

Comment:
This could significantly change the style of dropdowns, did you check that they look OK still? For example the CategorySelect component